src/valid/mod.rs

```diff
@@ -114,6 +114,8 @@ bitflags::bitflags! {
         const RAY_QUERY = 0x1000;
         /// Support for generating two sources for blending from fragement shaders
         const DUAL_SOURCE_BLENDING = 0x2000;
+        /// Support for binding arrays.
+        const BINDING_ARRAY = 0x4000;
```

Comment:
This will have to be split out eventually https://github.com/gfx-rs/naga/issues/2151. Should we do it now instead?